README.md

```diff
@@ -1,46 +1,429 @@
+
+### Setting up a Testnet with Arc Contracts
+
+Arc.js runs against an Ethereum network where it assumes that the Arc contracts have been migrated.  Out of the box, Arc.js can find contracts migrated to the mainnet. But for testing, you will need to tell it to migrate the Arc contracts to a testnet of your choice.  You can do this by running a few Arc.js scripts.
+
+For example, to deploy contracts to a ganache testnet, run the following scripts from the root of your application:
+
+On Linux and others [until runAsync is fixed for Linux](http://github.com/daostack/arc.js/issues/64), run this in a separate window of your own creation:
+
+```script
+npm explore @daostack/arc.js -- npm start test.ganache.run
+```
+
+On Windows: 
+```script
+npm explore @daostack/arc.js -- npm start test.ganache.runAsync
+```
+
+Now migrate the Arc contracts to ganache:
+
+```script
+npm explore @daostack/arc.js -- npm start migrateContracts
+```
+
+Now when your app uses Arc.js, it will be running against ganache and the contracts you just migrated.
 
+For more on Arc.js scripts, see [Working with Arc.js Scripts](#working-with-arcjs-scripts).
 
-# Arc Configuration
-The default configuration settings for Arc can be found in config/default.json, they are:
+### Using the Arc.js Library in your Code
+
+#### Importing Arc.js
+
+Import everything from ArcJs as follows:
 
 ```javascript
+import * as ArcJs from '@daostack/arc.js';
+```
+
+#### Configuring Arc.js
+The default configuration settings for Arc.js can be found in its `config/default.json` file. A few  examples:
+
+```json
 {
-  "providerUrl": "http://localhost:8545",
-  "network": "kovan", // Options are 'homestead', 'ropsten', 'rinkeby', 'kovan'
-  "gasLimit": [a number like 6015000]
+  "providerUrl": "http://127.0.0.0:8545",
+  "network": "ganache",
+  "gasLimit": 6015000
 }
 ```
 
-To get a configuration setting use:
+To obtain a configuration setting:
+
 ```javascript
 import { config } from '@daostack/arc.js';
 config.get('network');
 ```
 
-You can override configuration settings at runtime with:
+To override a configuration setting at runtime:
 ```javascript
 import { config } from '@daostack/arc.js';
-config.set('network', 'ropsten');
+config.set('network', 'kovan');
+```
+
+You can also override the default configuration settings by setting values on properties of `node.env` (see [here](http://nodejs.org/dist/latest-v9.x/docs/api/process.html#process_process_env)) with the same name as the corresponding arc.js configuration setting.  This enables you to use environment variables to control the arc.js configuration.
+
+#### Working with DAOs and Arc Contracts
+Now that you've got Arc.js plugged into your application and configured, and contracts migrated to a running testnet, you are ready to start coding against DAOs and Arc contracts.  The following sections describe how.
+
+One thing to remember:  All token and reputation amounts should be expressed in Wei.
+
+## Working with DAOs
```

Comment:
I am suggesting to move this section under docs somewhere..
and add a link to that .

There is too much content on the root readme.

Reply:
I responded elsewhere in the PR about splitting things out.  [Here for example](https://github.com/daostack/arc.js/pull/74#issuecomment-364263579).  That task needs to be a separate PR.